distribution/adapters/js-adapter-npm-zip/pom.xml

```diff
@@ -53,7 +53,7 @@
                             <includeGroupIds>org.keycloak</includeGroupIds>
                             <includeArtifactIds>keycloak-js-adapter</includeArtifactIds>
                             <outputDirectory>${project.build.directory}/unpacked/js-adapter</outputDirectory>
-                            <includes>*.js,*.map,*.d.ts</includes>
+                            <includes>*.js,*.map,*.d.ts,README.md</includes>
```

Comment:
I'm not sure if README.md is placed in the right place to be picked up. 

@stianst any advice here?

Reply:
I think you can include all `.md` files, when publishing NPM will filter out these files anyways. You'll also have to add this to both `pom.xml` and `assembly.xml` for the `js-adapter-npm-zip` and `js-adapter-zip` distributions.

I have another PR that modifies these files in the same manner here https://github.com/keycloak/keycloak/pull/9351.